src/game/boe.actions.cpp

```diff
@@ -2588,7 +2588,7 @@ void handle_death() {
 			fs::path file_to_load = nav_get_party();
 			if(!file_to_load.empty()) load_party(file_to_load, univ);
 			if(univ.party.is_alive()) {
-				if(overall_mode != MODE_STARTUP)
+				if(overall_mode == MODE_STARTUP)
```

Comment:
This doesn't look right. `post_load()` shows various scrollbars to indicate that you're in the game, so it shouldn't be called if mode is `MODE_STARTUP`.

Reply:
The current code sets overall_mode to MODE_STARTUP:
```
void handle_death() {
	std::string choice;
	
	overall_mode = MODE_STARTUP;
	
	while(true) {
..
			fs::path file_to_load = nav_get_party();
			if(!file_to_load.empty()) load_party(file_to_load, univ);
			if(univ.party.is_alive()) {
				if(overall_mode != MODE_STARTUP)
```
and if I don't make an error, load_party never changes overall_mode. So when a party is loaded (and alive), we always have overall_mode == MODE_STARTUP but we still have to finish loading the party.

Ie. Basically, this patch should make the code work, but this code should probably be completely rewritten.

Reply:
If I'm not mistaken, `load_party` never changes the mode, right? Or am I missing something?

Maybe the correct fix is this?

```cpp
				finish_load_party();
				if(overall_mode != MODE_STARTUP)
					post_load();
```

I'm not sure if swapping the order those are called in works though.

Reply:
Yes, it should work. Ie., post_load seems to mainly update the screen (and do some cleaning).